src/VisualStudio/VisualBasic/Impl/Options/IntelliSenseOptionPageControl.xaml

```diff
@@ -15,14 +15,20 @@
             <GroupBox x:Uid="CompletionListsGroupBox"
                       Header="{x:Static local:IntelliSenseOptionPageStrings.Option_CompletionLists}">
                 <StackPanel>
-
                     <CheckBox x:Uid="Highlight_matching_portions_of_completion_list_items"
                                   x:Name="Highlight_matching_portions_of_completion_list_items"
                                   Content="{x:Static local:IntelliSenseOptionPageStrings.Option_Highlight_matching_portions_of_completion_list_items}" />
                     <CheckBox x:Uid="Show_completion_item_filters"
                                   x:Name="Show_completion_item_filters"
                                   Content="{x:Static local:IntelliSenseOptionPageStrings.Option_Show_completion_item_filters}" />
-
+                </StackPanel>
+            </GroupBox>
+            <GroupBox x:Uid="SelectionInCompletionListGroupBox"
```

Comment:
i.e. more like this:

![image](https://cloud.githubusercontent.com/assets/4564579/15801170/907a4da4-2a41-11e6-99b6-e1151dc039b6.png)